imageroot/systemd/user/mariadb.service

```diff
@@ -32,8 +32,13 @@ ExecStart=/usr/bin/podman run \
 	--env=NETHCTI_DB_USER \
 	--env=NETHCTI_DB_PASSWORD \
 	--env=PHONEBOOK* \
-	--env=ASTERISK* \
+	--env=ASTERISK_RTPSTART \
+	--env=ASTERISK_RTPEND \
+	--env=ASTERISK_SIP_PORT \
+	--env=ASTERISK_SIPS_PORT \
+	--env=ASTERISK_IAX_PORT \
```

Comment:
Any reason why the vars has been expanded from previous `ASTERISK_*`?

Reply:
Just a matter of preference (and a forced manual conflict resolve), if a variable needs renaming or I need to find cases where my variable is used, `ASTERISK_*` obfuscates the search and allows access to the container to unforeseeable variables.